src/v0/destinations/pagerduty/data/pagerdutyChangeEventConfig.json

```diff
@@ -0,0 +1,33 @@
+[
+  {
```

Comment:
yes, so i think keeping separate files looks code more cleaner because keeping one file we need to delete some payload from changeEvent also source property is required for file and for other it's optional. let me know what do you think?

Reply:
I would call @ItsSudip to pitch in here.

Reply:
We can have separate files.